src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -116,25 +115,31 @@ private JsonErrorResult produceErrorResponse(String message, HttpStatus status,
         if (transformer != null) {
             message = transformer.transform(message);
         }
-        return new JsonErrorResult(status.name(), message);
+        return new JsonErrorResult(status, message);
     }
 
     @JsonSerialize(include = JsonSerialize.Inclusion.NON_EMPTY)
     static class JsonErrorResult {
-        private String error_code;
-        private String description;
+        public static final String ERROR_URN = "urn:ietf:params:scim:api:messages:2.0:Error";
```

Comment:
I realize that we have a [place](https://github.com/osiam/scim-schema/blob/master/src/main/java/org/osiam/resources/scim/Constants.java) where we keep our URNs. I will move it there once this PR, osiam/connector4java-integration-tests#289 and osiam/connector4java#156 are merged.